### DISCUSSION TITLE
Sorry for being despite.

### DISCUSSION THREAD

**Discussion by jorshua2784** [community/community#145086] [Discover: GitHub Best Practices]
I never imagine that I gone to grow a big community. My apologies. 

Now is a new beginning

**Comment by denis23x** [comment]
Hi @jorshua2784 

Do you need any help?

**Comment by jorshua2784** [comment]
Yes I need some help @Denis Iakimenko
I need to do few things but I get late from my job and it’s hard to me. I
want to do so many things but I need to do it.

*Jorshua Lopez*
Director de Ventas y Servicio
JLR Distributors L.L.C.


On Thu, Nov 21, 2024 at 1:07 PM Denis Iakimenko ***@***.***>
wroth

**Comment by ebndev** [comment]
Hi @jorshua2784, thanks for being a part of the GitHub Community!

In our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md), we ask that members contribute in a positive and constructive way. This type of post is considered "off topic" as GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. Any repeat and future violations may result in a temporary or indefinite block from the Community. 

We appreciate your understanding.
